docs/user/tutorials/01-40-expose-workload/01-40-expose-workload-apigateway.md

```diff
@@ -7,13 +7,38 @@ This tutorial shows how to expose an unsecured instance of the HTTPBin Service a
 
 ## Prerequisites
 
-* Deploy [a sample HTTPBin Service](../01-00-create-workload.md).
-* Set up [your custom domain](../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead. 
-* Depending on whether you use your custom domain or a Kyma domain, export the necessary values as environment variables:
+* [Deploy a sample HTTPBin Service](../01-00-create-workload.md).
+* [Set up your custom domain](../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead. 
+
+## Steps
+
+<!-- tabs:start -->
+#### **Kyma Dashboard**
+
+1. Go to **Discovery and Network > API Rules** and select **Create**. 
+2. Provide the following configuration details.
+  - **Name**: `httpbin`
+  - In the `Service` section, select:
+      - **Service Name**: `httpbin`
+      - **Port**: `8000`
+  - To fill in the `Gateway` section, use these values:
+    - **Namespace** is the name of the namespace in which you deployed an instance of the HTTPBin Service. If you use a Kyma domain, select the `kyma-system` namespace.
+    - **Name** is the Gateway's name. If you use a Kyma domain, select `kyma-gateway`. 
+    - In the **Host** field, enter `httpbin.{DOMAIN_TO_EXPORT_WORKLOADS}`. Replace the placeholder with the name of your domain.
+  - In the `Rules` section, select:
```

Comment:
This configuration is different from the one in the kubectl. here we only configure one rule. If we want to have exactly the same configuration, we can simplify the configuration in the kubectl section, but this should also have no influence on the functionality.

Reply:
I've adjusted the Kyma Dashboard section